charts/images.yaml

```diff
@@ -11,6 +11,14 @@ images:
   sourceRepository: https://github.com/metal-stack/machine-controller-manager-provider-metal
   repository: ghcr.io/metal-stack/machine-controller-manager-provider-metal
   tag: "v0.1.5"
+- name: auditforwarder
+  sourceRepository: https://github.com/mreiger/audit-forwarder
```

Comment:
We can move this to the metal-stack organization.

Reply:
Moved as suggested; it runs in test.